app/controllers/courses_controller.rb

```diff
@@ -75,7 +90,7 @@ def publish
         ]
       }.to_h
     else
-      flash[:success] = 'Your changes have been published'
+      flash[:success] = "Your course has been published. The link for this course is: https://localhost:5000/course/#{@provider.provider_code}/#{@course.course_code}"
```

Comment:
This flash message ought to have "Your course has been published" as the title, and the following sentence as a paragraph. But this can come in a follow up PR as it'll involve refactoring our flash messages somewhat.